docs/setup.rst

```diff
@@ -0,0 +1,95 @@
+Setting Up Corobo
+=================
+
+The following documentation illustrates the process to
+install corobo and interface it with the Gitter 
+(or any other backend) API.
+
+Installation
+------------
+- To install corobo, first you need to install 
+  python (version 3.6 or more) and set up a virtual environment. 
+  This can be done as follows:
+  ::
+    python3 -m venv env
+    cd env
+    source bin/activate ---->  For Linux
+    \Scripts\activate   ---->  For Windows
```

Comment:
This could be a bit confusing, let's not specify instructions for windows or other OS.